app/src/main/java/com/github/se/cyrcle/model/parking/ParkingViewModel.kt

```diff
@@ -121,6 +121,11 @@ class ParkingViewModel(
     return parkingRepository.getNewUid()
   }
 
+  fun updateReviewScore(newScore: Double, parking: Parking) {
```

Comment:
Since the parking is selected when the user decides to review or edit , is it necessary to put the parking as argument instead of using `selectedParking`

Reply:
Good fix with the default argument and well done to have updated the doc to specify the condition to use the default argument. Since we will mainly use this function to add a review the default argument is good way of simplifying the use of the function by others developers .